src/notebook/widget.ts

```diff
@@ -337,6 +337,25 @@ class NotebookWidget extends Widget {
 }
 
 
+function createElements(tag: string, attr?: any, children?: HTMLElement[] | string): HTMLElement {
```

Comment:
How about this compromise, and leave other attributes for later:

`createElement(tag: string, className: string, children?: HTMLElement[])`